src/main/java/task/Deadline.java

```diff
@@ -0,0 +1,21 @@
+package task;
+
+public class Deadline extends Task {
+    protected String by;
```

Comment:
Why is this protected?
Are you inheriting anything else from deadline to make use of this field?